connectivity/FEATURE_BLE/source/cordio/source/GattServerImpl.cpp

```diff
@@ -1720,13 +1729,18 @@ void GattServer::handleEvent(
     GattAttribute::Handle_t attributeHandle
 )
 {
+    // To be used in cases where the characteristic value handle differs from the attribute handle
+    GattAttribute::Handle_t charHandle;
+
     switch (type) {
         case GattServerEvents::GATT_EVENT_UPDATES_ENABLED:
             tr_info("Updates enabled for attribute %d on connection %d", attributeHandle, connHandle);
+            get_value_handle_by_cccd_handle(attributeHandle, charHandle);
```

Comment:
Please check the return of the function.

Reply:
I don't think it's required. `get_value_handle_by_cccd_handle` will return false if and only if there's no index for the given handle. Unless I'm mistaken, that's impossible here.

Reply:
you can always just make an assert

Reply:
It is unlikely indeed. However it depends on how well this is checked by the Cordio stack. An assert wouldn't hurt.

Reply:
Done: 52483f3e30e38447f8fbe81afafdbab1c1b566d2